include/range/v3/view/linear_distribute.hpp

```diff
@@ -71,15 +75,16 @@ namespace ranges
                 }
                 else
                 {
-                    from_ += (to_ - from_) / T(n_);
+                    from_ = T(to_ - (delta_ * (n_ - 1)));
```

Comment:
Maybe?

```suggestion
                    from_ = T(to_ - (delta_ * Calc(n_ - 1)));
```

Reply:
Makes sense - thank you for pointing that out.